tests/desktop/test_desktop_api_browse.py

```
@@ -0,0 +1,120 @@
+        mock_webview.FOLDER_DIALOG = 1  # constant from pywebview
+        return mock_webview, mock_window
+
+    # ------------------------------------------------------------------
+    # Happy path
+    # ------------------------------------------------------------------
+
+    def test_returns_first_path_when_user_selects(self) -> None:
+        """browse_directory() must return the first element of the result tuple."""
+        from file_organizer.desktop.app import DesktopAPI
+
+        mock_webview, mock_window = self._make_mock_webview(("/Users/rahul/Documents",))
```

Ruff (0.15.9)

[warning] Unpacked variable `mock_window` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)